Added Rviz configuration file (relates to #2)

KoksalBerkay opened the pull request:
Added a base config file for camera image displays. Since I don’t have physical access to the cameras, I couldn’t verify the topic names directly. The topics used are placeholders, and updating them to match reseq_ros2 should be straightforward.

KoksalBerkay committed:
Added Rviz configuration file (relates to #2)

KoksalBerkay committed:
Camera topic names corrected

luca-byte reviewed (changes requested):
Discussed in person: now we know the topics 🎯

KoksalBerkay commented:
I have changed the topic names correctly now. Also fixed the problem where image windows changed their order every time RViz was opened.

> Discussed in person: now we know the topics 🎯

KoksalBerkay committed:
created the launch file (relates to #4)
